W-12996402 Update create-custom-namespace.adoc

bilalbox opened the pull request:
Fixed to update behavior seen in RTF 2.2.5 release

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

bilalbox committed:
Update create-custom-namespace.adoc

Fixed to update behavior seen in RTF 2.2.5 release

hpanchal1221 commented on modules/ROOT/pages/create-custom-namespace.adoc:
```diff
@@ -12,13 +12,14 @@ When you deploy an application into an Anypoint Platform environment, Runtime Fa
 ----
 kubectl create ns <YOUR_ENVIRONMENT_NAME>
-kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>
+kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/agentNamespace=rtf rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID> rtf.mulesoft.com/org=<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>
```
Instead of `rtf` we should probably be adding a placeholder for the namespace `<rtf_namespace>`

hpanchal1221 commented on modules/ROOT/pages/create-custom-namespace.adoc:
```diff
@@ -12,13 +12,14 @@ When you deploy an application into an Anypoint Platform environment, Runtime Fa
 * Replace `<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>` with the correct ID for your environment.
-
-After you run these commands, any application deployed into an Anypoint Platform environment identified by `<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>` is created inside the `<YOUR_ENVIRONMENT_NAME>` namespace.
+* Replace `<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>` with the correct ID for your Business Group.
```
And we can add the description for the place holder here: Replace `<rtf_namespace>` with the correct agent namespace to associate with.

hpanchal1221 commented on modules/ROOT/pages/create-custom-namespace.adoc:
```diff
@@ -12,13 +12,14 @@ When you deploy an application into an Anypoint Platform environment, Runtime Fa
+kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/agentNamespace=rtf rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID> rtf.mulesoft.com/org=<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>
 ----
 +
 * Replace `<YOUR_ENVIRONMENT_NAME>` with the name of your namespace.
```
Can we also change all the upper cases for placeholders with lower cases so that this will be consistent throughout. 
`<YOUR_ENVIRONMENT_NAME>` to `<app_namespace>`
`<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>` to `<environment_id>`
`<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>` to `<org_id>`

bilalbox replied:
It was all uppercase in the original document, but yeah maybe moving them all lowercase will be a better look.

bilalbox replied:
After trying it all with lowercase, if find that those parameters really pop with the caps. I think we should keep it.

bilalbox committed:
Update create-custom-namespace.adoc

Implemented suggestion to parameterize the RTF namespace to cover cases where there is are multiple RTF instances in one cluster, or at least a non-default RTF namespace is used

bilalbox reviewed (commented):
Give it another look and let me know if this passes muster. Thanks for your cooperation!

hannanelson committed:
Merge branch 'v2.2' into patch-1

kevintroller committed:
Updated namespaces to lowercase
